package.json

```diff
@@ -17,6 +17,7 @@
     "es6-error": "^4.1.1",
     "fuse.js": "^3.4.5",
     "history": "^4.9.0",
+    "open": "6.4.0",
```

Comment:
these changes to `package.json` seem unintentional? I don't see `open` used anywhere in your code.

Reply:
Should be gone now.  I was having issues with WSL and needed to manually set the `open` version to get the site running on my machine.